apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.test.js

```diff
@@ -56,6 +56,10 @@ function generateQRCode (qrCodeData = {}, { version = '0001', encodingTag = '2'
         ...qrCodeData,
     }
 
+    // Should work case-insensitively
```

Comment:
??? why don't want to change qrCodeObj ?

Reply:
Because `PersAcc` may be passed within `qrCodeData`.

Reply:
Fixed. The separated case-insensitive test was created.